tests/discord-alert-sanitization.unit.test.ts

```
@@ -0,0 +1,139 @@
+import { afterEach, describe, expect, it, vi } from 'vitest'
+import { sendDiscordAlert500 } from '../supabase/functions/_backend/utils/discord.ts'
+import {
+  sanitizeSensitiveFromString,
+  sanitizeSensitiveHeaders,
+} from '../supabase/functions/_backend/utils/discord_sanitization.ts'
+
+function sampleValue(prefix: string) {
+  return `${prefix}-${'x'.repeat(16)}`
+}
+
+describe('Discord alert sanitization', () => {
```

ESLint

[error] `describe`s should begin with lowercase

(test/prefer-lowercase-title)